MAUI/Calendar/customizations.md

```diff
@@ -222,4 +293,346 @@ You can customize the selection cell background and text style in Month, Year, D
+
+You can customize the month cell appearance by using the `CellTemplate` property of `YearView` in the `SfCalendar`. The `DataTemplateSelector` for a calendar can be used to choose a different data template for each cell in the calendar based on the cell's data. It allows customizing the appearance of a particular cell based on certain conditions.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+    <Grid.Resources>
+        <DataTemplate x:Key="normalDateTemplate">
+            <Grid Background = "Pink" >
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Year}"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="todayDateTemplate">
+            <Grid Background = "PaleGreen">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Year}"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="leadingDateTemplate">
+            <Grid Background = "Purple">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Year}"/>
+            </Grid>
+        </DataTemplate>
+        <local:MonthCellTemplateSelector x:Key="monthCellTemplateSelector" 
+                                         TodayDateTemplate="{StaticResource todayDateTemplate}" 
+                                         NormalDateTemplate="{StaticResource normalDateTemplate}" 
+                                         LeadingDateTemplate="{StaticResource leadingDateTemplate}"/>
+    </Grid.Resources>
+    <calendar:SfCalendar x:Name="Calendar" 
+                        View="Decade" >
+        <calendar:SfCalendar.YearView>
+            <calendar:CalendarYearView CellTemplate="{StaticResource yearCellTemplateSelector}" />
+        </calendar:SfCalendar.YearView>
+    </calendar:SfCalendar>
+    </Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="YearCellTemplateSelector.cs" %}
+
+public class YearCellTemplateSelector : DataTemplateSelector
+{
+    public YearCellTemplateSelector()
+    {
+    }
+    public DataTemplate NormalDateTemplate { get; set; }
+    public DataTemplate TodayDateTemplate { get; set; }
+    public DataTemplate LeadingDateTemplate { get; set; }
+
+    protected override DataTemplate OnSelectTemplate(object item, BindableObject container)
+    {
+        var yearCellDetails = item as CalendarCellDetails;
+        if (yearCellDetails.Year == DateTime.Today.Year)
+            return TodayDateTemplate;
+        else if (yearCellDetails.IsTrailingOrLeadingDate)
+            return LeadingDateTemplate;
+        else
+            return NormalDateTemplate;
+    }
+}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Decade;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Cell temaplte in .NET MAUI Calendar.](images/customization/maui-cell_template.png)
```

Comment:
This images are not relatable for above code, remove the images

Reply:
The mentioned images are removed.